### ISSUE TITLE
Any active fork out there?

### ISSUE THREAD

**Issue by fmikker** [Grive/grive#277]
Hi,
Since this project hasn't been updated since 2013, I'm wondering if there is any fork out there that IS being updated with all the patches people are sending in?

**Comment by jpschewe** [comment]
@linwiz @MasterSoft24 I see both of you have mentioned that you have active forks of grive. Can we pick one to be the new source for grive, short of getting write access to this repository? 

I am using grive-tools, https://launchpad.net/~thefanclub/+archive/ubuntu/grive-tools, under Ubuntu. I haven't yet determined which git repository they are pulling from. I sent them an email today asking this. I'd like to have a better grive application under grive-tools that doesn't loose my data when folders are renamed on Google, #258.

**Comment by jpschewe** [comment]
I received a reply from the Ubuntu group and they are using this repository for distribution.

**Comment by MasterSoft24** [comment]
Yes it is. I fork this repository to develop more advanced GUI client on Qt.

**Comment by jpschewe** [comment]
@MasterSoft24 Have you merged in the pull requests from this repository? Or are you primarily focusing on the GUI?

**Comment by linwiz** [comment]
I'm not developing grive myself, I just put together a fork that merged in other commits. I've become dissatisfied with grive and don't use it myself.

**Comment by jpschewe** [comment]
What are you using for Google Drive syncing on Linux now? Or did you just
give up?

On Thu, Jan 8, 2015 at 11:07 AM, Matthew Motz <notifications@github.com>
wrote:

> I'm not developing grive myself, I just put together a fork that merged in
> other commits. I've become dissatisfied with grive and don't use it myself.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Grive/grive/issues/277#issuecomment-69211777>.
>



-- 
http://mtu.net/~jpschewe

**Comment by mattmorgan2** [comment]
I just installed InSync first thing today, and paid for it 
(https://insynchq.com). I think if I was on my own I could deal with the 
major bug in Grive (you know, that it deletes entire folder trees when 
you move them in Drive), but I use Linux at work and the team shares a 
folder structure--so I'd probably end up deleting other people's files 
without even knowing about it.

Insync is $15 once per account for life (or there are business licenses 
with other schemes). I have not used it long enough to give a 
comprehensive recommendation, but they assure me it does not have that 
troublesome bug, and the install/config & payment processes were very 
nicely done. It seems to be updating all by itself pretty successfully 
so far (but that's only like 3 hrs of work).

**Comment by linwiz** [comment]
I don't have a syncing solution for google drive for linux, I do everything the old fashion way, uploading to the website.

**Comment by jpschewe** [comment]
@linwiz So your fork isn't going to be updated, correct?


On Thu, Jan 8, 2015 at 12:14 PM, Matthew Motz <notifications@github.com>
wrote:

> I don't have a syncing solution for google drive for linux, I do
> everything the old fashion way, uploading to the website.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Grive/grive/issues/277#issuecomment-69222586>.
>



-- 
http://mtu.net/~jpschewe

**Comment by linwiz** [comment]
Not actively updated, but I will makes changes as I discover commits that are produced. I think Grive2 looks promising though I haven't used it.
